backend/prisma/schema.prisma

```diff
@@ -18,3 +31,44 @@ model User {
   failedLogins Int     @default(0)
   verified     Boolean @default(false)
 }
+
+model Category {
+  id        Int        @id @default(autoincrement())
+  name      String     @unique
+  questions Question[]
+}
+
+model Topic {
+  id        Int        @id @default(autoincrement())
+  name      String     @unique
+  questions Question[]
+}
+
+model SolutionTemplate {
+  id         Int      @id @default(autoincrement())
+  language   Language
+  template   String
+  question   Question @relation(fields: [questionId], references: [id])
+  questionId Int
+}
+
+model TestCases {
+  id         Int      @id @default(autoincrement())
+  inputs     String[]
+  output     String
+  question   Question @relation(fields: [questionId], references: [id])
+  questionId Int
+}
+
+model Question {
+  id         Int                @id @default(autoincrement())
+  title      String             @unique
+  difficulty Difficulty
+  category   Category           @relation(fields: [categoryId], references: [id])
+  categoryId Int
+  topics     Topic[]
+  question   String
```

Comment:
Can we call this questionContent or content? Other models are referencing the Question table via a question field so it may create slight confusion

Reply:
I think we can call it `description` instead then.